pkg/adsc/delta.go

```diff
@@ -196,6 +205,7 @@ type Client struct {
 
 	// initialWatches is the list of resources we are watching on startup
 	initialWatches []resourceKey
+	sync           map[string]time.Time
```

Comment:
what's this

Reply:
like adsc, used to record which type of config has synced, which is needed by the config controller.